root/static/styles/layout.less

```diff
@@ -169,7 +169,7 @@ pre code {
                 float: right;
                 form {
                     width: 302px;
-                    margin-top: 3px;
+                    margin-top: -1px;
```

Comment:
The suggestion was made [here](https://github.com/metabrainz/musicbrainz-server/pull/229#discussion_r52579134), where it was actually `-2px`. However, I tested (on FF and Chromium) and `-1px` looks better IMO.